components/server/src/auth/authenticator.ts

```diff
@@ -136,6 +140,9 @@ export class Authenticator {
         });
         // authenticate user
         authProvider.authorize(req, res, next);
+        // TODO (arthursens): the line below is wrong!
+        // We are increasing the succesful counter even if authorization failed
+        increaseLoginCounter("succeeded", authProvider.info.host)
```

Comment:
the single way to succeed a login is at https://github.com/gitpod-io/gitpod/blob/master/components/server/src/auth/login-completion-handler.ts#L75